Classes/Base/Config.php

```diff
@@ -52,10 +52,12 @@ public static function get( array $paths, array $local = array() )
 			self::$config = $conf;
 		}
 
-		if( isset( $local['typo3']['tsconfig'] ) )
+		// Check for array
+		if( isset( $local['typo3']['tsconfig'] ) && is_array( $local['typo3']['tsconfig'] ) )
```

Comment:
On my tests I saw that the $local['typo3']['tsconfig'] can be set but not to an array. I this case it makes no sense to parse it, only a timeloss. So I added the additional check for an array